.travis.yml

```diff
@@ -1,76 +1,68 @@
-language: python
+language: c
 
-python:
-  - 2.7
+os:
+    - linux
+    - osx
 
-virtualenv:
-     system_site_packages: false
+cache:
+    directories:
+        $HOME/miniconda/pkgs
+        /Users/travis/miniconda3/pkgs
```

Comment:
One is the Mac path and one is the linux one, it is the default install location for miniconda. These directives do not actually do anything at the moment.